src/klein/test/test_resource.py

```diff
@@ -30,14 +31,14 @@
 
 
 def requestMock(
-    path,
-    method=b"GET",
-    host=b"localhost",
-    port=8080,
-    isSecure=False,
-    body=None,
-    headers=None,
-):
+    path: bytes,
+    method: bytes = b"GET",
+    host: bytes = b"localhost",
+    port: int = 8080,
+    isSecure: bool = False,
+    body: Optional[bytes] = None,
```

Comment:
Would this work ? 

```suggestion
    body: bytes = b"",
```

This is just trying to remove the usage of `None`.

Reply:
Yeah I was debating between changing the code or typing it as-is. Since `requestMock` has callers elsewhere I'll give this a go.